tasks/eth/ink-003-fp-holocene-pectra-upgrade/.env

```diff
@@ -7,10 +7,11 @@ SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=18
 SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=16
 
 #Expected address
-EXPECTED_PERMISSIONED_DISPUTE_GAME_IMPL=0xf75DA262a580cf9b2EaB9847559c5837381aE3a3
-EXPECTED_PERMISSIONED_DELAYED_WETH_PROXY=0x95367c3d0fEa41BD95E6E710A680Cc75d99db31F
-EXPECTED_FAULT_DISPUTE_GAME_IMPL=0xE09185Bbb538d084209EF6f1a92Dc499D313ce51
-EXPECTED_DELAYED_WETH_PROXY=0xC6707118240626bd361F0c90C44E202BcD0A24c8
+
+EXPECTED_PERMISSIONED_DISPUTE_GAME_IMPL=0x8D9faaEb46cBCf487baf2182E438Ac3D0847F637
+EXPECTED_PERMISSIONED_DELAYED_WETH_PROXY=0xc4986627A41cdCf6fa33543D96f00F475bCE42f5
```

Comment:
I need to investigate but each time we create a new DG we need to deploy a WETH proxy right?

Reply:
For the most part that is correct, the DGs can be configured to use existing WETH contracts, but in practice that really hasn't happened because op-challenger handles the different WETH contracts dynamically.